development/buildkit-compose.yaml

```diff
@@ -15,7 +15,7 @@
 version: "3.8"
 services:
   buildkitd:
-    image: moby/buildkit:master-rootless
+    image: moby/buildkit:v0.8.1-rootless
```

Comment:
What was the reason for moving from `master` tag to `v0.8.1`? Is it because the `master` image is updated often and might include breaking changes? Is this something that you recommend doing on our Prowjobs as well?

Reply:
master is the dev build off the head of the master branch. Yeh, we should probably do that for Prow.

Reply:
Gotcha, so we would want to use the latest stable version from DockerHub instead of dev builds.